config/database.yml

```diff
@@ -17,6 +17,8 @@
 default: &default
   adapter: postgresql
   encoding: unicode
+  username: postgres
```

Comment:
do you need it here? Maybe you can't see problem here. But imagine if multiple developers work on the same project. They all need to change `database.yml` everytime they need to run an app.